packages/cli/src/commands/job.ts

```diff
@@ -0,0 +1,75 @@
+import ora from 'ora';
+import { runJob, getJobById, getJobByPipeline, getJobs, getLogById } from '../service/job';
+
+const PipelineStatus: any = {
+  0: 'creating',
+  1: 'running',
+  2: 'success',
+  3: 'fail'
+};
+
+const spinner = ora();
+
+export const job = async (operation: string, id: string, pipelineId: string, verbose: string) => {
+  let data: any;
+  switch (operation) {
+  case 'run':
+    if (!pipelineId) {
+      spinner.fail(`Please provide the pipeline id to run`);
+      return;
+    }
+    data = await runJob(pipelineId);
+    spinner.succeed(`create job ${data.id} succeeded`);
+    if (verbose === 'true') {
+      let logs = ' ';
+      const timer = setInterval(async () => {
```

Comment:
How about use recursion setTimeout() to replace setInterval() because of the sync callback of this setInterval. 
Sync callback in setInterval is unsafe:(